jitsi/env.example

```diff
@@ -0,0 +1,92 @@
+#-----------------------------------------------------------------------
+# IMPORTANT: NEVER commit the actual .env file to version control!
+#-----------------------------------------------------------------------
+
+#-----------------------------------------------------------------------
+# DOCKER CONFIGURATION
+#-----------------------------------------------------------------------
+# Jitsi component versions (stable, latest, or specific version)
+JITSI_VERSION=stable
+
+# Container restart policy (unless-stopped, always, on-failure)
+RESTART_POLICY=unless-stopped
+
+# Optional: Custom container names
+# PROSODY_CONTAINER_NAME=jitsi_prosody
+# JICOFO_CONTAINER_NAME=jitsi_jicofo
+# JVB_CONTAINER_NAME=jitsi_jvb
+# WEB_CONTAINER_NAME=jitsi_web
+
+# Web interface ports
+HTTP_PORT=8000
+HTTPS_PORT=8443
+
+#-----------------------------------------------------------------------
+# BASIC SETUP
+#-----------------------------------------------------------------------
+# Base URL of your Jitsi instance (must match your domain)
+PUBLIC_URL=https://jitsi.test.mydomain.de
```

Comment:
"jitsi.test.mydomain.de" appears several times in this config, it would potentially be easier to extract it into a variable and the reuse that variable, or?